src/status_im2/contexts/communities/discover/view.cljs

```diff
@@ -22,23 +21,26 @@
                              :group [{:id         1
                                       :token-icon (resources/get-mock-image :status-logo)}]}]}})
 
-(defn render-fn
-  [community-item _ _ {:keys [width view-type]}]
-  (let [item  (merge community-item
-                     (get mock-community-item-data :data))
-        cover {:uri (get-in (:images item) [:banner :uri])}]
+(defn community-list-item
+  [item _ _ {:keys [width view-type ids?]}]
+  (let [community (merge
+                   (if ids?
+                     (rf/sub [:communities/home-item item])
```

Comment:
@jo-mut, the following is just a suggestion :)

It seems we can remove the workaround where the `community-list-item` function needs to deal with a sequence of community IDs or a sequence of communities and the whole usage of the `ids?` flag. I think it's cleaner if the callers that use `community-list-item` fetch all the community maps/objects if they're just IDs.

In the code below, notice the new subscription `:communities/from-ids`. This subscription would also encapsulate the logic to use mock data or not, something that's not so good from the current `community-list-item` function, since it knows too much (it knows if the data should be mocked or not). As you know, `featured-list` would also need a similar change.

```clojure
(defn community-list-item
  [{:keys [id] :as community} _ _ {:keys [width view-type]}]
  (let [cover {:uri (get-in (:images community) [:banner :uri])}]
    (if (= view-type :card-view)
      [quo/community-card-view-item (assoc community :width width :cover cover)
       #(rf/dispatch [:navigate-to :community-overview id])]
      [quo/communities-list-view-item
       {:on-press      (fn []
                         (rf/dispatch [:communities/load-category-states id])
                         (rf/dispatch [:dismiss-keyboard])
                         (rf/dispatch [:navigate-to :community {:community-id id}]))
        :on-long-press #(rf/dispatch
                         [:bottom-sheet/show-sheet
                          {:content (fn [] [options/community-options-bottom-sheet id])}])}])))

(defn other-communities-list
  [{:keys [communities communities-ids view-type ids?]}]
  (let [communities (if ids?
                      (rf/sub [:communities/from-ids communities-ids])
                      communities)]
    [rn/view style/other-communities-container
     [rn/flat-list
      {:key-fn                       :id
       :keyboard-should-persist-taps :always
       :separator                    [rn/view {:height 16}]
       :data                         communities
       :render-fn                    community-list-item
       :contentContainerStyle        style/flat-list-container
       :render-data                  {:view-type view-type}}]]))
```